package-lock.json

```diff
@@ -4,6 +4,24 @@
     "lockfileVersion": 1,
     "requires": true,
     "dependencies": {
+        "@azure/cosmos": {
```

Comment:
Turns out this was added in https://github.com/microsoft/vscode-cosmosdb/pull/1424, but package-lock.json was never updated

Reply:
Changed Dev Ops to use `npm ci` to prevent this problem. See this build as an example: https://dev.azure.com/ms-azuretools/AzCode/_build/results?buildId=18710&view=results